components/org.wso2.carbon.identity.recovery.endpoint/pom.xml

```diff
@@ -24,6 +24,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-war-plugin</artifactId>
+                <version>3.3.1</version>
```

Comment:
move the version to parent pom file and don't need to specify in the child pom files.
Fix in other places too